dotcom-rendering/src/components/LoopVideoPlayer.tsx

```diff
@@ -165,16 +165,26 @@ export const LoopVideoPlayer = forwardRef(
 			subtitleSource,
 			subtitleSize,
 			activeCue,
+			isCinemagraph,
 		}: Props,
 		ref: React.ForwardedRef<HTMLVideoElement>,
 	) => {
 		const loopVideoId = `loop-video-${uniqueId}`;
+		const showSubtitles =
+			!!subtitleSource &&
+			!!subtitleSize &&
+			!!activeCue?.text &&
+			!isCinemagraph;
```

Comment:
Do we need to check isCinemagraph here given we dont pass subtitleSource or subtitleSize the level above if it isCinemagraph?